### ISSUE TITLE
 Information tooltip which is appearing on focus is not conveyed to screen reader.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13754]
### Steps to reproduce:
1. Open URL https://www.bing.com/webmasters on New Edge Browser.
2. Login using valid credentials.
3. Navigate to side navigation bar.
4. Navigate to Tools & Enhancements and Expand it.
5. Navigate to Robots.Txt Tester and select it.
6. Turn on NVDA and the (i) icon appearing using tab key.
7. Observe that Information appearing which is appearing on focus is not conveyed to screen reader.

### Actual behavior:
Information tooltip which is appearing on focus is not conveyed to screen reader.

### Expected behavior:
Content which is visually displayed must also be available to assistive technology users. If the content is dynamic (such as an error message), focus management and/or aria-live can expose it to a screen reader. If the content is static, be sure it is not hidden with CSS or using aria-hidden.

**SME Recommendation**
Ensure that the popup is not hidden to screen readers. Screen reader users should be able to navigate inside of the popup when it is open. 

#### NVDA installed/portable/running from source:
OS: Windows_11
Version: 21H2
OS Build: 22000.194
Browser: New edge (96.0.1054.29)
URL: https://www.bing.com/webmasters 
Screen reader: NVDA 2020.4

#### NVDA version:
NVDA 2020.4

#### Windows version:
OS: Windows_11
Version: 21H2
OS Build: 22000.194

**Screen shot**
<img width="491" alt="image" src="https://user-images.githubusercontent.com/93735775/171134582-76d1bf8f-b7d7-49e5-9812-8c6469e29f48.png">

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
